server/routes/controllers/applications.js

```diff
@@ -0,0 +1,72 @@
+import bindFunctions from '../../lib/bindFunctions'
+import ApplicationModel from '../../database/models/Application'
+import mailer from '../../lib/mailer'
+import { NotFoundError } from '../../lib/errors'
+
+export default {
+  _init (Applications = ApplicationModel) {
+    bindFunctions(this)
+
+    this.Applications = Applications
+    return this
+  },
+
+  async getApplications (req, res) {
```

Comment:
for the query I would also put the state. It's not relevant for now, but in the future I think it can help.

Reply:
or is it possible by passing the project object and status is the only prop filled?

Reply:
check my latest commit below. is that what you are referring to?

Reply:
yep :)